ui/embed.go

```diff
@@ -0,0 +1,23 @@
+// +build !noViewer go1.16
```

Comment:
Then shouldn't this be `!noViewer,go1.16`?

Reply:
Good catch. 
I have now completely omitted the +built constraint in embed.go
It doesn't seem to matter because "embed.go" is only referred in "viewer_embed.go" which is only considered when built with go 1.16+
It compiled fine with no tag set and with -tag NoViewer on Go1.15.14.